contrib/Dockerfile.builder

```diff
@@ -33,11 +35,16 @@ RUN apt-get -qq update && \
 	wget && \
 	rm -rf /var/lib/apt/lists/*
 
+ENV LANGUAGE=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8
+RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales
+
 RUN cd /tmp/ && \
     wget https://bitcoin.org/bin/bitcoin-core-$BITCOIN_VERSION/bitcoin-$BITCOIN_VERSION-x86_64-linux-gnu.tar.gz -O bitcoin.tar.gz && \
     tar -xvzf bitcoin.tar.gz && \
     mv /tmp/bitcoin-$BITCOIN_VERSION/bin/bitcoin* /usr/local/bin/ && \
     rm -rf bitcoin.tar.gz /tmp/bitcoin-$BITCOIN_VERSION
 
 RUN pip3 install --upgrade pip && \
-    pip3 install python-bitcoinlib==0.7.0 pytest==3.0.5 setuptools==36.6.0 pytest-test-groups==1.0.3 flake8==3.5.0 pytest-rerunfailures==3.1
+    python3 -m pip install python-bitcoinlib==0.7.0 pytest==3.0.5 setuptools==36.6.0 pytest-test-groups==1.0.3 flake8==3.5.0 pytest-rerunfailures==3.1
```

Comment:
You change `s/pip3/python -m pip/` on this line, but not on the previous line?  This is deliberate?

Reply:
There is an outstanding issue that breaks the `pip3` wrapper after the upgrade. Hence the `pip3 install --upgrade pip` works, but afterwards `pip3` will fail due to the upgrade itself :-S